plugin-client/src/main/java/com/cloudera/nav/plugin/client/UpdatedResults.java

```diff
@@ -0,0 +1,46 @@
+package com.cloudera.nav.plugin.client;
+
+import java.util.Iterator;
+import java.util.Map;
+
+/**Class for clients to get all updated Entities and Relations and a marker of most recent extractorRunId's for each source,
+ * that can then be used in determining future incremental updates.
+ *
+ * Created by Nadia.Wallace on 6/4/15.
+ */
+public class UpdatedResults {
```

Comment:
Let's call this something like NavResultSet (use the refactor functionality in your IDE)